packages/contracts/source/contracts/crypto/networks.ts

```diff
@@ -43,12 +44,20 @@ export type MilestoneTimeouts = {
 	stageTimeoutIncrease: number;
 };
 
+export type MilestoneEvm = {
+	minimumGasLimit: number;
+	minimumGasFee: number;
+	nativeFeeMultiplier: number;
+	nativeGasLimits: Record<string, number>;
+};
+
 export type Milestone = {
 	height: number;
 	activeValidators: number;
 	address: Record<string, any>;
 	block: MilestoneBlock;
 	epoch: string;
+	evm: MilestoneEvm;
```

Comment:
Maybe evm -> gas, since gas is not only limited to evm, but is part to block and transactions.